lib/kennel/syncer.rb

```diff
@@ -164,14 +167,14 @@ def initialize
         @attribute_differ = AttributeDiffer.new
       end
 
-      def display(create, update, delete)
+      def display(internal_plan)
         Kennel.out.puts "Plan:"
-        if create.empty? && update.empty? && delete.empty?
+        if internal_plan.values.all?(&:empty?)
```

Comment:
```suggestion
        if noop?
```

Reply:
It's in a different class now 😅

Reply:
then it should have the `noop?` method ?